src/components/molecules/Post/Post.jsx

```diff
@@ -0,0 +1,65 @@
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+// import { Form } from 'formik';
+import { Text, ImageComponent } from '../..';
+import { routes } from '../../../routes/Routes';
+import { Icon } from '../../atoms/IconComponent/Icon';
+import { dateFormat } from '../../../utils/format';
+
+import './Post.scss';
+
+export const Post = ({ children, date, name, avatar, profileID }) => {
+    const navigate = useNavigate();
+    const redirect = useCallback(() => {
+        navigate(routes.Profile);
+    }, [navigate]);
+
+    return (
+        <div className='post__field'>
+            <ImageComponent
+                src={avatar}
+                size='small'
+                onClick={useCallback(() => {
+                    redirect(`${routes.Feed}${routes.Profile}/${profileID}`);
+                }, [profileID, redirect])}
+            />
+            <Text customClass='username' type='accent' size='medium'>
+                {name}
+            </Text>
+            <Text size='small' customClass='date' type='primary'>
+                {dateFormat(date)}
+            </Text>
+            <Text type='primary' customClass='message' size='medium'>
+                {children}
+            </Text>
+            <Icon
+                iconName='like'
+                // onClick={}
```

Comment:
remove any commented code pls

Reply:
It should be there, but it should be related to the counter next to the icons for which I had no idea / time, so in order not to block it, I just left it. Finish it or do we do a separate task for it? I guess it is not that important and I don't know if we can make it